src/brs/BlockchainProcessorImpl.java

```diff
@@ -948,17 +947,6 @@ private void pushBlock(final Block block) throws BlockNotAcceptedException {
         long calculatedTotalFee = 0;
         MessageDigest digest = Crypto.sha256();
 
-        ArrayList<Long> accountIds = new ArrayList<>();
-        block.getTransactions().forEach(t -> {
-          if (t.getRecipientId() != 0L)
-            accountIds.add(t.getRecipientId());
-          if (t.getSenderId() != 0L)
-            accountIds.add(t.getSenderId());
-        });
-        if (!accountIds.isEmpty()) {
-          stores.getAccountStore().getAccountTable().fillCache(accountIds);
-        }
```

Comment:
we can just lose this, no issues?

Reply:
The way this works is tries to predict what account ids will be needed and load them into cache ahead of time. However, this is guaranteed to be slower, even with perfect prediction (which it doesn't have), than caching on demand (when you retrieve the account)

Example: You need to use 2 accounts 2 times each. You cache them ahead of time and then each time you use them get them from the cache. You have done 2 database hits and 4 cache hits.

Now say we don't cache ahead of time and cache on demand. The first time you try to get each account they are not cached, so you get from db and store in cache. The second time you just hit the cache. You have done 2 database hits and 2 cache hits.

This alone means it's unnecessary, not even adding that the caching prediction isn't exactly perfect.